### ISSUE TITLE
Build on Windows wrong OS

### ISSUE THREAD

**Issue by Lervanov** [vergecurrency/VERGE#173]
compiler": "gcc",
  "env": "VERGE_PLATFORM='windows'",
  "file": "release_windows.zip",
  "os": "linux",
  "sudo": "required"

The OS should be Windows, not linux. I noticed this error a few hours ago and it didn't let windows build compile, i think this might cause it

**Comment by Lervanov** [comment]
https://travis-ci.org/vergecurrency/VERGE/builds/323562341

Build jobs

Windows has Linux icon, that's how I figured it out. The description contains that linux OS mention

**Comment by Lervanov** [comment]
I don't know if you have a bounty system, but I want to try my luck

DBHdPnbyaPhaGRjVfd545GfekM98SWLtgq

You guys are doing a fantastic job and I've been on this train since an interview i saw with Justin a few months ago. I believe Verge can easily spread and become the de facto privacy coin in the next 6 months. 

Happy new year, guys!

**Comment by justinvforvendetta** [comment]
what do you mean a bounty? did you make a pull request for something?

**Comment by Lervanov** [comment]
No, i am typing this from a fairly limited feature phone (i'm on vacation) and have no access to any laptop, let alone code editor. I can browse the web and wanted to let you guys know!

**Comment by davecummins2000** [comment]
![image](https://user-images.githubusercontent.com/34986750/34463106-63bf75b8-ee4a-11e7-9d92-ee32978bfedd.png)

https://github.com/vergecurrency/VERGE/blob/master/.travis.yml

This needs line 14 to be windows not linux

**Comment by Ronan-Farrell** [comment]
Nice spot @Lervanov
A more professional and mature OSS team would say thanks for the input. Contributions and inclusion from the community should be welcomed and encouraged.

**Comment by Lervanov** [comment]
Thanks Dave, i can't take screenshots on my phone and couldn't exactly point out. That's exactly what i meant.

**Comment by triamazikamno** [comment]
There is nothing wrong, they are cross-compiling windows binary in the linux container
